unofficial_flocker_tools/config.py

```diff
@@ -143,6 +144,36 @@ def main(reactor, args):
 
     yield gatherResults(deferreds)
 
+    if c.config["os"] == "ubuntu":
+        if len(sys.argv) > 2:
+            if sys.argv[2] == "--swarm":
+                clusterid = c.runSSH(c.config["control_node"], """
+docker run swarm create""").strip()
+                log("Created Swarm ID")
+                for node in c.config["agent_nodes"]:
+                    d = c.runSSHAsync(node['public'], """
+service docker stop
+docker daemon -H unix:///var/run/docker.sock -H tcp://0.0.0.0:2375 >> /tmp/dockerlogs 2>&1 &
```

Comment:
agreed, that is the way I ultimately wanted to do it. For sake of time I did it thise way but agree setting the options in default/docker its better. thanks will work on this.